contrib/mpi-proxy-split/mpi-wrappers/mpi_file_wrappers.cpp

```diff
@@ -0,0 +1,91 @@
+/****************************************************************************
+ *   Copyright (C) 2019-2021 by Gene Cooperman, Rohan Garg, Yao Xu          *
+ *   gene@ccs.neu.edu, rohgarg@ccs.neu.edu, xu.yao1@northeastern.edu        *
```

Comment:
Illio,
    The convention is that each file should mention only the names of authors who made significant changes to the file.  Please don't hesitate to remove me, Rohan, and Yao, and replace all three with your own name.

In addition, for other files, if you just changed 2 or 3 lines, it's not important.  But where you made significant changes to a file (perhaps a large function, or 2 or more small functions), then please add your name to the copyright, in alphabetical order.